src/js/server/signup.js

```diff
@@ -108,7 +71,7 @@ fluid.defaults("gpii.express.user.signup.post.handler", {
         },
         "checkForExistingUser": {
             funcName: "gpii.express.user.signup.post.handler.checkForExistingUser",
-            args:     ["{that}", "{arguments}.0"]
+            args:     ["{that}", "{gpii.express.user.utils}", "{arguments}.0"]
```

Comment:
`{arguments}.0` is no longer needed here.

Reply:
I believe this one is still necessary as it's doing a check for the existence of the username before moving forward.